app/scripts/constants.js

```diff
@@ -40,5 +40,142 @@ window.OPENSHIFT_CONSTANTS = {
    
       // These are things like DCPs that aren't actually persisted resources
       "DeploymentConfigRollback"
+  ],
+
+  // href's will be prefixed with /project/{{projectName}} unless they are absolute URLs
+  PROJECT_NAVIGATION: [
```

Comment:
Yep although looking at this and seeing my comment above, I forgot to actually handle the absolute URLs